.changeset/big-pots-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
+---
+
+Crud-Generator: Remove unnecessary await for delete mutation
```

Comment:
```suggestion
API Generator: Remove unnecessary await for delete mutation
```

We normally use API Generator or Admin Generator instead of CRUD Generator

Reply:
Done with d23731b4ee62ed56f895d709693252a98c438e95